go.mod

```diff
@@ -0,0 +1,26 @@
+module github.com/opendevstack/ods-core
```

Comment:
Given that we already have a Go module in jenkins/webhook-proxy I don't really like this being at the root. Can we move it elsewhere?

Reply:
Moved